python/py-tensorflow/Portfile

```diff
@@ -64,7 +58,7 @@ if {${name} ne ${subport}} {
         port:py${python.version}-mock \
         port:py${python.version}-backports.weakref
 
-    if {${python.version} < 34} {
+    if {${python.version} < 35} {
```

Comment:
This was more correct before (even if you dropped the `34` subport). If someone accidentally adds 34 back, the dependencies will be wrong/nonexistent.

Reply:
Agree with @mojca

Reply:
You're right. Thanks, I'll fix it.

Reply:
@mojca @pmetzger I've fixed it and squashed.

Reply:
I've fixed it.